packages/constants/src/fields.ts

```diff
@@ -30,6 +30,8 @@ export const TYPES = [
 	'unknown',
 ] as const;
 
+export const NUMERIC_TYPES = ['bigInteger', 'decimal', 'float', 'integer'] as const;
```

Comment:
Should we call it `NUMBER_TYPES` since `numeric` is already an alias for `decimal` in SQL?